src/storage/kv.rs

```diff
@@ -303,41 +351,41 @@ impl LogKeyValueStore {
                     appeared_key.insert(key.clone());
 
                     if target_key == key {
-                        return Some((instruction, ChainHeight::new((index) as u64)));
+                        Some((instruction, ChainHeight::new((index) as u64)))
                     } else {
-                        return None;
+                        None
                     }
                 }
                 Instruction::RevertOne { key, height: _ } => {
                     appeared_key.insert(key.clone());
 
                     if target_key == key {
-                        return Some((instruction, ChainHeight::new((index) as u64)));
+                        Some((instruction, ChainHeight::new((index) as u64)))
                     } else {
-                        return None;
+                        None
                     }
                 }
                 Instruction::RevertAll { height: _ } => {
                     if appeared_key.contains(&target_key) {
-                        return Some((instruction, ChainHeight::new((index) as u64)));
+                        Some((instruction, ChainHeight::new((index) as u64)))
                     } else {
-                        return None;
+                        None
                     }
                 }
                 Instruction::RemoveOne { key } => {
                     appeared_key.insert(key.clone());
 
                     if target_key == key {
-                        return Some((instruction, ChainHeight::new((index) as u64)));
+                        Some((instruction, ChainHeight::new((index) as u64)))
                     } else {
-                        return None;
+                        None
                     }
                 }
                 Instruction::RemoveAll => {
                     if appeared_key.contains(&target_key) {
-                        return Some((instruction, ChainHeight::new((index) as u64)));
+                        Some((instruction, ChainHeight::new((index) as u64)))
                     } else {
-                        return None;
```

Comment:
Formatting changes strike again. I think they should stay but it's a bit annoying.

Reply:
ref to previous comments.